### ISSUE TITLE
TUN/TAP Interface Unused/unusable?

### ISSUE THREAD

**Issue by ghost** [des-testbed/libara#84]
Currently, LibARA-testbed is not setup to send packets into libdessert via TUN/TAP. The documentation of libdessert covers quite well how to deal with packets received via TUN/TAP, but how does one sent data TO the TUN/TAP interface to be encapsulated in a packet?

If TUN/TAP is to remain unused, then the toNetwork callback should simply drop the packet, since no data being sent from user space through TUN/TAP is actually LibARA data- I think it might be a good idea to circumvent this interface to eliminate system overhead packets (remember the 6 mysterious packets sent over mesh interface when we start the daemon?).

TL;DR we are still using Dessert to access the NIC, simply not to send/receive data from LibARA.
